### PR TITLE
Add move view command

### PR DESCRIPTION
**Pull request by frkwy**
@flyingmutant 

PR of `move view quicky with document.`

:)

### CODE REVIEW

File: command_mode.c [hunk 1]
```diff
@@ -1295,6 +1295,39 @@ static void cmd_prev_view(char *arg)
...
+        }
+    }
+}
+
+static void cmd_right_view(char *arg)
+{
+    if (cur_view == HELP_VIEW) {
+        set_view(TREE_VIEW);
+        prev_view = HELP_VIEW;
+
```
Comment (flyingmutant) [inline command_mode.c]: Extra empty line

**Comment by flyingmutant** [comment]
Please view tabs, not spaces for indentation, like the rest of cmus code.

File: Doc/cmus.txt [hunk 2]
```diff
@@ -476,6 +476,12 @@ player-stop (*v*)
 prev-view
 	Go to previously used view.
 
+left-view
+        Go from current view to left view.
+
+right-view
+        Go from current view to right view.
```
Comment (flyingmutant) [inline Doc/cmus.txt]: I think this description is unclear, please explain in greater detail.

**Comment by frkwy** [comment]
@flyingmutant 
Thanks you for review.
I fixed indentation.

and I reconsider a document.

**Comment by frkwy** [comment]
Fixed a pointed item. Could you again review?

**Comment by flyingmutant** [comment]
Merged in e31988d728b46baa34f284a477b69023bfc9835a (and greatly simplified in db0f8a54c563cfba0e4cf060117de9e85134a8c6), thanks!

**Pull request closed without merging**